### PR TITLE
test: add tests for onfido-php library

### PR DESCRIPTION
**Pull request by sofia-gomes-onfido**
Includes tests for the following instances:
- applicants
- address
- webhooks
- checks
- documents
- extraction
- live photos
- live videos
- motion
- reports (including detailed validation of schemas)
- SDK token
- tasks
- workflow runs (including detailed validation of output schemas)
- watchlist monitors

### CODE REVIEW

File: test/Resource/ChecksTest.php
Comment (sofia-gomes-onfido) [reply test/Resource/ChecksTest.php:88]: It is, if a test doesn't have an assertion, phpunit will consider it `risky`. 
I did some digging on how to best way to handle a test that doesn't throw any exceptions, since phpunit doesn't have the equivalent of junit's `pass()`. The recommendation was to do that assertion.

I forgot to add the reasoning in a comment.

**Review by dvacca-onfido — APPROVED** [review]
LGTM, I'm just wondering if we should squash all these commit before merging or if it makes sense keeping each of them aside.

**Pull request merged**
